catalyst-gateway/bin/src/db/types/mod.rs

```diff
@@ -0,0 +1,15 @@
+//! Wrappers over commonly used types that can be stored to and load from a database.
+
+pub use slot::DbSlot;
```

Comment:
Do we need this DbType 🤔

Reply:
I can don't mind removing it if we decide so, but I think that the following types are confusing:
```rust
type PrimaryKey = (Vec<u8>, Vec<u8>, num_bigint::BigInt, i16, bool);
type PrimaryKey = (Vec<u8>, num_bigint::BigInt, i16, i16, Vec<u8>, Vec<u8>);
```
It isn't clear what these fields represent. Proper types can make reading such code easier and modifying it less error prone.